src/python/WMCore/Services/ReqMgr/ReqMgr.py

```diff
@@ -72,6 +72,16 @@ def _createQuery(self, queryDict):
 
         return args.rstrip('&')
 
+    def getParentLocks(self):
+        """
+        _getParentLocks_
```

Comment:
How about elaborating on this docstring? Specifying the return format is also useful IMO.

Reply:
This  was  method created _Ad exemplum_ :  
https://github.com/dmwm/WMCore/blob/a3e1e6b623a8d8dcbafd381d252f1215658f21ad/src/python/WMCore/Services/WMStatsServer/WMStatsServer.py#L80-L86

Reply:
You got it.
Done